README.md

```diff
@@ -46,6 +46,28 @@ $ kubectl apply -f https://raw.githubusercontent.com/mittwald/kubernetes-replica
 
 ## Usage
 
+### Role and rolebinding replication
+
+In order to replicate roles and rolebindings, the service account tied to the replicator needs to have the correct permissions. The chart currently offers two options to accomplish this:
```

Comment:
Wow, this is a good catch. 🤩  Maybe we can document this even more verbosely, to avoid any further confusion: To create a new role, your own account needs to have at least the same set of privileges as the role you're trying to create.